egs/swbd/s5c/local/reverberate/prep_sim_rirs.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+# Copyright 2016  Tom Ko
+# Apache 2.0
+# script to simulate room impulse responses (RIRs)
+
+sampling_rate=8000
```

Comment:
An option is to generate a 'sufficient' number of RIRs ourselves and put them in a Kaldi-readable format on openslr.org, and then just have the script that generated them provided for information.